domain/src/main/java/starterProject/App.java

```diff
@@ -15,19 +15,17 @@ public String getGreeting() {
     public static void main(String[] args) {
         createCar();
     }
-
-    private static List<Car> createCar() {
+    private static void createCar() {
```

Comment:
This method doesn't really do much, why was it changed to be void?

Reply:
I was getting a warning from inteliJ saying the return value is never used, and that i should make it a void, thus i made the method a void.